templates/aws-stack.yml

```diff
@@ -944,12 +944,6 @@ Resources:
     Condition: CreateSecretsBucket
     DeletionPolicy: Retain
     Properties:
-      AccessControl: LogDeliveryWrite
-      PublicAccessBlockConfiguration:
-        BlockPublicAcls: true
-        BlockPublicPolicy: true
-        IgnorePublicAcls: true
-        RestrictPublicBuckets: true
```

Comment:
It also looks like the UI still shows a scary warning message if you don't have this:
![2023-04-24T01:23:23,118602840+10:00](https://user-images.githubusercontent.com/11096602/233848690-f6de2d5f-e0cb-4ece-b672-fcd259163806.png)

So I think we should undelete the `PublicAccessBlockConfiguration`

Reply:
@triarius Yeah, it seems that not all defaults have been applied yet! It's likely still rolling out, as the expectation was to have these being rolled out as default